chromium_src/chrome/browser/component_updater/registration.cc

```diff
@@ -33,6 +37,10 @@ void RegisterComponentsForUpdate() {
   ComponentUpdateService* cus = g_browser_process->component_updater();
   brave_wallet::RegisterWalletDataFilesComponent(cus);
   brave_shields::RegisterHTTPSEverywhereComponent(cus);
+  if (base::FeatureList::IsEnabled(psst::features::kBravePsst)) {
```

Comment:
same as above, the feature check should go in `RegisterPsstComponent`, but `Maybe` doesn't seem to be the convention here for some reason so just keep the existing method name.